ansible/pbTestScripts/qemuPlaybookCheck.sh

```diff
@@ -237,11 +237,29 @@ done
      	  $EXTRA_ARGS \
 	  -nographic) > "$workFolder/${OS}.${ARCHITECTURE}.startlog" 2>&1 &
 
-	echo "Machine is booting; logging console to $workFolder/${OS}.${ARCHITECTURE}.startlog Please be patient"
-	sleep 180
-	tail "$workFolder/${OS}.${ARCHITECTURE}.startlog" | sed 's/^/CONSOLE > /g'
-	echo "Machine has started, unless the above log shows otherwise ..."
+	echo "Machine is booting; logging console to $workFolder/${OS}.${ARCHITECTURE}.startlog"
+	echo "Please be patient"
```

Comment:
Make it clear how long the timeout is (300 assumes you also take my suggestion below, otherwise make it 180!)

```suggestion
	echo "Please be patient - this can take up to 300 seconds"
```

Reply:
Fixed with the latest force push :-)